content/en/auto-scaling.md

```diff
@@ -2,7 +2,7 @@
 title: Autoscaling
 status: Completed
 category: property
-tags: ["application", "infrastructure", "architecture"]
+tags: ["property", "infrastructure", "architecture"]
```

Comment:
Should we remove architecture? 

Reminder: for these to be effective, we wanted to restrict them to the most relevant tags. A less is more approach, basically.

Reply:
Agree :)